lib/pages/child/task_in_progress_page.dart

```diff
@@ -0,0 +1,323 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fokus/logic/timer/timer_cubit.dart';
+import 'package:fokus/services/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
+import 'package:fokus/widgets/buttons/double_circular_notched_button.dart';
+import 'package:fokus/widgets/cards/item_card.dart';
+import 'package:fokus/widgets/cards/sliding_card.dart';
+import 'package:fokus/widgets/chips/attribute_chip.dart';
+import 'package:fokus/widgets/large_timer.dart';
+import 'package:fokus/widgets/task_app_header.dart';
+import 'package:lottie/lottie.dart';
+
+enum TaskInProgressPageState {currentlyCompleting, inBreak, done, rejected}
+
+class ChildTaskInProgressPage extends StatefulWidget {
+  @override
+  _ChildTaskInProgressPageState createState() => _ChildTaskInProgressPageState();
+}
+
+class _ChildTaskInProgressPageState extends State<ChildTaskInProgressPage> with TickerProviderStateMixin {
+  final String _pageKey = 'page.childSection.taskInProgress';
+  TaskInProgressPageState _state = TaskInProgressPageState.currentlyCompleting;
+	Animation<Offset> _offsetAnimation;
+	Animation<Offset> _taskListFabAnimation;
+	AnimationController _bottomBarController;
+	final GlobalKey<SlidingCardState> breakCard = GlobalKey<SlidingCardState>();
+	final GlobalKey<SlidingCardState> completingCard = GlobalKey<SlidingCardState>();
+	final GlobalKey<SlidingCardState> rejectCard = GlobalKey<SlidingCardState>();
+	final GlobalKey<SlidingCardState> doneCard = GlobalKey<SlidingCardState>();
+	final GlobalKey<TaskAppHeaderState> header = GlobalKey<TaskAppHeaderState>();
+
+
+
+	@override
+  Widget build(BuildContext context) {
+  	return Scaffold(
+			appBar: TaskAppHeader(
+				height: 330,
+				title: '$_pageKey.header.title',
+				appHeaderWidget: ItemCard(
+					title: "Sprzątanie pokoju",
+					subtitle: "Co każdy poniedziałek, środę, czwartek i piątek",
+					isActive: true,
+					progressPercentage: 0.4,
+					activeProgressBarColor: AppColors.childActionColor,
+					textMaxLines: 2,
+					chips:
+					<Widget>[
+						AttributeChip.withIcon(
+							icon: Icons.description,
+							color: AppColors.childBackgroundColor,
+							content: AppLocales.of(context).translate('page.childSection.panel.content.taskProgress', {'NUM_TASKS': 2, 'NUM_ALL_TASKS': 5})
+						)
+					],
+				),
+				helpPage: 'plan_info',
+				key: header,
+				breakPerformingTransition: _breakPerformingTransition,
+				isBreak: false
+			),
+			body: Stack(
+				children: [
+					SlidingCard(
```

Comment:
As there are 4 of those cards and they are quite similar you could extract them to function and parameterize the color, animation and content?